pybikes/encicla.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2015, Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+import codecs
+
+from lxml import etree
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils, exceptions
+
+__all__ = ['Encicla', 'EnciclaStation']
+
+parse_methods = {
+  'json': 'get_json_stations',
+}
+
+class Encicla(BikeShareSystem):
+
+  sync = True
+
+  meta = {
+    'system': 'Encicla',
+    'company': 'Sistema de Bicicletas Públicas del Valle de Aburrá'
+  }
+
+  def __init__(self, tag, feed_url, meta, format):
+    super(Encicla, self).__init__(tag, meta)
+    self.feed_url = feed_url
+    self.method = format
+
+  def update(self, scraper = None):
+    if scraper is None:
+      scraper = utils.PyBikesScraper()
+
+    if self.method not in parse_methods:
+      raise Exception(
+        'Extractor for method %s is not implemented' % self.method )
+
+    self.stations = eval(parse_methods[self.method])(self, scraper)
```

Comment:
Same here as before. In bixi this made some sense. Here it would be just abusing the style too much